specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4073,6 +4073,14 @@
         }
       }
     },
+    "AzureFirstPartyManagedCertificate": {
+      "description": "Azure FirstParty Managed Certificate provided by other first party rp's, used for https",
```

Comment:
nit : Can we say "Azure First Party Managed Certificate provided by other first party RP's to enable HTTPS."